doc/user/layouts/partials/create-source/connector/s3/details.html

```diff
@@ -1,72 +1,70 @@
   you can only entirely replace or delete objects.
 
 The primary effect of this is that we do not handle *updates* to S3 objects, and we may interleave
 multiple object ingestion to speed it up.
 
 [aws-s3]: https://aws.amazon.com/s3/
 
-#### S3 Limitations
+#### S3 limitations
 
 * Currently S3 sources do not support Avro- or Protobuf- encoded objects. Implementation issues:
   [Avro](https://github.com/MaterializeInc/materialize/issues/5596),
   [Protobuf](https://github.com/MaterializeInc/materialize/issues/5597)
 * Object ingest order is not guaranteed.
-* **All S3 sources are append-only** -- deleted and updated [S3 Objects][] are silently ignored.
+* **All S3 sources are append-only**. Deleted and updated [S3 objects][] are silently ignored.
 
 [S3 Objects]: https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingObjects.html
 
 #### Object discovery strategies
 
 Materialize has several techniques to discover the objects to download, with [more
-planned][object-specifiers] for the future. Strategies may be specified multiple times within one
-**CREATE SOURCE** statement, so for example this is a legal invocation: `DISCOVER OBJECTS USING BUCKET SCAN
-'example', BUCKET SCAN 'other', SQS NOTIFICATIONS 'example-notifications'`.
+planned][object-specifiers] for the future.
 
-All strategies follow the same basic pattern, `materialized` will:
+All strategies follow the same basic pattern.  Materialize will:
 
-* Interact with a collection of AWS APIs.
-* Deduplicate objects such that the same object is never downloaded twice.
-* Filter the list of objects discovered via those APIs against the [**MATCHING** clause
+* Obtain a list of objects using AWS APIs.
+* Deduplicate objects so the same object is never downloaded twice.
+* Filter the list of objects against the [**MATCHING** clause
   patterns](#patterns).
+* Download the matching objects.
 * Treat each object downloaded as a newline-delimited file for the purposes of record
   delimitation.
 
 No guarantees are made about the relative order of records sent from the object discovery process
-and the Materialize SQL engine, as usual with SQL you must impose your own order.
+and the Materialize SQL engine. As usual with SQL, you must impose your own order.
 
-[object-specifies]: https://github.com/MaterializeInc/materialize/issues/5502
+[object-specifiers]: https://github.com/MaterializeInc/materialize/issues/5502
 
-##### Scanning S3 Buckets
+You may specify strategies multiple times within one
+`CREATE SOURCE` statement. For example, this is a legal invocation:
 
-The **DISCOVER OBJECTS USING BUCKET SCAN** syntax causes S3 sources to issue [ListBucket][] requests to
-collect the names of objects to collect. After collecting the list of keys, their names are
-checked against the **MATCHING** clause, if present, before downloading and extracting objects
-line by line.
+```
+DISCOVER OBJECTS USING BUCKET SCAN 'example',
+BUCKET SCAN 'other', SQS NOTIFICATIONS 'example-notifications'
+```
```

Comment:
nit: IMO the fact that the first `BUCKET SCAN` is on the same line as the opening clause but the others are not makes the point of this example slightly obscured, this might be clearer:

```suggestion
```sql
DISCOVER OBJECTS USING
    BUCKET SCAN 'example',
    BUCKET SCAN 'other',
    SQS NOTIFICATIONS 'example-notifications'
```
```

otherwise I'd advocate for putting all object specifiers on the same line, either the same as `DISCOVER OBJECTS` or the next line.